### ISSUE TITLE
Create Anna for firefox

### ISSUE THREAD

**Issue by gauthamzz** [Anna-Assistant/Anna#140]
Almost all code is same, I guess it wont be that hard. 

Once its done comment, ill make a seperate repo for that.

**Comment by ananya** [comment]
I will try just try

**Comment by gauthamzz** [comment]
@ananya-agrawal update?

**Comment by ananya** [comment]
I searched for it and will try fixing it . If somebody else wants to take the issue he can

**Comment by srvkmr130** [comment]
I will try too.

**Comment by savannahar68** [comment]
@gauthamzz Hello sir,i read the code and have understood all,im implementing this into firefox,
as the speech reorganization feature was added on 20th Feb by Firefox(beta) so will take a day more.
Assign it to me
Thank you.

**Comment by gauthamzz** [comment]
Use https://developer.mozilla.org/en-US/Add-ons/WebExtensions webextension. 
I plan to port it :smile:

**Comment by savannahar68** [comment]
@gauthamzz  Sir, after spending 2 days on it, I come to the final conclusion about porting Anna to firefox.
Firefox has not yet released there webspeech API and they are planning to do it soon in Q2 of 2018(according to official blog post)
PS- This issue has been unresolved for 4 years now - [BugZilla LInk](https://bugzilla.mozilla.org/show_bug.cgi?id=1248897) 
So we can hold until firefox launches the API, until then nothing can be done, i tried with everything else but there is no external library support for web extension of firefox.

**Comment by gauthamzz** [comment]
you can port from chrome extersnion to webextensino which will support all browsers using the existing code. just minor changes.

**Comment by savannahar68** [comment]
@gauthamzz Yes sir,i i know that, but there is not Speechrecognization Api like one in chrome in firefox !!! so we cant implement Anna in firefox until then .

**Comment by gauthamzz** [comment]
https://github.com/TalAter/annyang/blob/master/src/annyang.js#L33

Do this

**Comment by savannahar68** [comment]
Sir, already had a hand on it, Firefox currently doesn't support speechrecog, so that library won't work

![image](https://user-images.githubusercontent.com/20986069/37056201-ac7d80f0-21a9-11e8-9476-add9922a3da4.png)

When opened in Firefox it gives error

**Comment by gauthamzz** [comment]
okay. lets wait then :smile:

**Comment by savannahar68** [comment]
The only option left :p
